presidio-analyzer/install_dependencies.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+poetry install --no-interaction
+
+if [ "$DEV_MODE" == "transformers" ]; then
+  poetry add torch transformers huggingface_hub
```

Comment:
I think that this can be moved the the pyproject.toml extra dependencies, and you can make that argument a cli argument / extra dependencies argument

Reply:
Okay take a look at the new version